plugins/pickdeep.yaml

```diff
@@ -0,0 +1,53 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pickdeep
+spec:
+  version: v0.0.10-alpha
+  platforms:
+    - bin: kupid.exe
+      uri: https://github.com/flavono123/kupid/releases/download/v0.0.10-alpha/pickdeep_v0.0.10-alpha_windows_arm64.zip
+      sha256: 09be7cae288d67d811f2a3bcc87a13f19f5d9dfb535e492a149c4f0991a5dbc9
+      selector:
+        matchLabels:
+          os: windows
+          arch: arm64
+    - bin: kupid.exe
+      uri: https://github.com/flavono123/kupid/releases/download/v0.0.10-alpha/pickdeep_v0.0.10-alpha_windows_amd64.zip
+      sha256: d0dec95056ce0754bdc71fd82235d4f522dc744eb1bbf9c3956b2a4538a69104
+      selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+    - bin: kupid
+      uri: https://github.com/flavono123/kupid/releases/download/v0.0.10-alpha/pickdeep_v0.0.10-alpha_linux_arm64.tar.gz
+      sha256: e9253664874c2a5831543587147fdd8e08bbfb890cf74d1e7f1abbbac9727b16
+      selector:
+        matchLabels:
+          os: linux
+          arch: arm64
+    - bin: kupid
+      uri: https://github.com/flavono123/kupid/releases/download/v0.0.10-alpha/pickdeep_v0.0.10-alpha_linux_amd64.tar.gz
+      sha256: 6eade6170a48688b5b791e46a21277dfab3c4be7727b04215e1bcbeb7de83a63
+      selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+    - bin: kupid
+      uri: https://github.com/flavono123/kupid/releases/download/v0.0.10-alpha/pickdeep_v0.0.10-alpha_darwin_arm64.tar.gz
+      sha256: 4049602de4b44e7439319b71c183c2b474f84a57e12b718d472698aea767a7bd
+      selector:
+        matchLabels:
+          os: darwin
+          arch: arm64
+    - bin: kupid
+      uri: https://github.com/flavono123/kupid/releases/download/v0.0.10-alpha/pickdeep_v0.0.10-alpha_darwin_amd64.tar.gz
+      sha256: 4a9928fe48926cfd00336a9cc8bc07c4b8a863fce2c1c1bda7bfe674f9377c48
+      selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+  shortDescription: Pick any Deep schema fields, create your own table view of Kubernetes TUI.
+  homepage: https://github.com/flavono123/kupid
+  description: Pick any Deep schema fields, create your own table view of Kubernetes TUI.
```

Comment:
What do you think about something more explanatory for these descriptions:

Short: Interactively choose get output columns

Long: This is an interactive tool to pick which resource fields you want to show in the kubectl get output.

Reply:
agreed. and the short one will be represented in https://krew.sigs.k8s.io/plugins/ here right?